frontend-react/src/pages/Daily.tsx

```diff
@@ -5,6 +5,9 @@ import OrganizationResource from "../resources/OrganizationResource";
 import { useOktaAuth } from "@okta/okta-react";
 import { groupToOrg } from "../webreceiver-utils";
 import download from "downloadjs";
+import { getListOfSenders } from "../controllers/ReportController";
+import { ButtonGroup, Button } from "@trussworks/react-uswds"
+import { useState } from "react";
 
 const TableData = (props) => {
```

Comment:
Not part of this PR, but it would be nice to get a type other than `*any` here for the props at some point. 
(e.g. below, is `props.report.content` required or could report be `null`?)

Reply:
I wouldn't be against refactoring our `src/pages` files a bit; we also have some duplicated components like `<ReportLink>` which exists in `Daily.tsx` and `Details.tsx`. During that process, I'll add Prop types to each component.

Reply:
Sounds like a great plan.